tests/Admin/ArticleCategoryAdminTest.php

```diff
@@ -0,0 +1,104 @@
+        $this->assertStatusCode(Response::HTTP_OK, $this->client);
+        $form = $crawler->selectButton('btn_create_and_edit')->form();
+        $this->client->submit($form);
+
+        $this->assertStatusCode(Response::HTTP_OK, $this->client);
+        $this->assertValidationErrors(['data.name', 'data.slug'], $this->client->getContainer());
+    }
+
+    public function testCreateCategoryWithoutCTASuccess()
+    {
+        $this->authenticateAsAdmin($this->client, 'superadmin@en-marche-dev.fr', 'superadmin');
+
+        $crawler = $this->client->request('GET', $this->getUrl('admin_app_articlecategory_create'));
+        $this->assertStatusCode(Response::HTTP_OK, $this->client);
+        $form = $crawler->selectButton('btn_create_and_edit')->form();
+
+        $prefix = $this->getPrefixAdminForm($form);
+        $form->setValues([$prefix.'[name]' => 'Opinion', $prefix.'[slug]' => 'opinion', $prefix.'[position]' => 10]);
+        $this->client->submit($form);
+
+        $this->assertStatusCode(Response::HTTP_FOUND, $this->client);
+    }
+
+    public function testCreateCategoryWithCTASuccess()
+    {
+        $this->authenticateAsAdmin($this->client, 'superadmin@en-marche-dev.fr', 'superadmin');
+
+        $crawler = $this->client->request('GET', $this->getUrl('admin_app_articlecategory_create'));
+        $this->assertStatusCode(Response::HTTP_OK, $this->client);
+        $form = $crawler->selectButton('btn_create_and_edit')->form();
+
+        $prefix = $this->getPrefixAdminForm($form);
+        $form->setValues([
+            $prefix.'[name]' => 'Opinion',
+            $prefix.'[slug]' => 'opinion',
+            $prefix.'[position]' => 10,
+            $prefix.'[ctaLink]' => 'http://www.google.fr',
+            $prefix.'[ctaLabel]' => 'Google link',
+        ]);
+        $this->client->submit($form);
+
+        $this->assertStatusCode(Response::HTTP_FOUND, $this->client);
+    }
+
+    public function testCreateCategoryWithCTAFail()
+    {
+        $this->authenticateAsAdmin($this->client, 'superadmin@en-marche-dev.fr', 'superadmin');
+
+        $crawler = $this->client->request('GET', $this->getUrl('admin_app_articlecategory_create'));
+        $this->assertStatusCode(Response::HTTP_OK, $this->client);
+        $form = $crawler->selectButton('btn_create_and_edit')->form();
+
+        $prefix = $this->getPrefixAdminForm($form);
+        $form->setValues([
+            $prefix.'[name]' => 'Opinion',
+            $prefix.'[slug]' => 'opinion',
+            $prefix.'[position]' => 10,
+            $prefix.'[ctaLink]' => 'not a link',
+            $prefix.'[ctaLabel]' => 'Google link',
+        ]);
+        $this->client->submit($form);
+
+        $this->assertStatusCode(Response::HTTP_OK, $this->client);
+        $this->assertValidationErrors(['data.ctaLink'], $this->client->getContainer());
+    }
+
+    protected function getPrefixAdminForm(Form $form): ?string
```

Comment:
Please move this helper method to the trait, this could be useful in other tests. e.g.: 

https://github.com/EnMarche/en-marche.fr/blob/bb2f53988be179046d1cdcd130dc2d4cb86eb5af/tests/Admin/AdherentAdminTest.php#L42-L46
https://github.com/EnMarche/en-marche.fr/blob/bb2f53988be179046d1cdcd130dc2d4cb86eb5af/tests/Controller/Admin/AdminTimelineMeasureControllerTest.php#L103-L107

Also, regarding the method name and the regex pattern: this method isn't specific to admin forms.